concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -1,909 +1,909 @@
+Marrison-1967-909-493	493	mouth	674	MOUTH	2
+Marrison-1967-909-494	494	move	765	MOVE	2
+Marrison-1967-909-495	495	much	531	HOW MUCH	2
+Marrison-1967-909-496	496	mud	640	MUD	2
+Marrison-1967-909-497	497	must	971	MUST	2
+Marrison-1967-909-498	498	mustard			
+Marrison-1967-909-499	499	naked	590	NAKED	2
+Marrison-1967-909-500	500	name	1405	NAME	2
+Marrison-1967-909-501	501	narrow	1267	NARROW	2
+Marrison-1967-909-502	502	navel	1838	NAVEL	2
+Marrison-1967-909-503	503	near	1942	NEAR	2
+Marrison-1967-909-504	504	necessary	1030	NECESSARY	2
+Marrison-1967-909-505	505	neck	1333	NECK	2
+Marrison-1967-909-506	506	necklace	1961	NECKLACE	2
+Marrison-1967-909-507	507	needle	1382	NEEDLE (FOR SEWING)	2
+Marrison-1967-909-508	508	neighbor	1332	NEIGHBOUR	2
+Marrison-1967-909-509	509	nest	539	NEST	2
+Marrison-1967-909-510	510	net	2963	NET	2
+Marrison-1967-909-511	511	never	1656	NEVER	2
+Marrison-1967-909-512	512	new	1231	NEW	2
+Marrison-1967-909-513	513	news	2964	NEWS	2
+Marrison-1967-909-514	514	night	1233	NIGHT	2
+Marrison-1967-909-515	515	no	1269	NO	2
+Marrison-1967-909-516	516	noise	1182	NOISE	2
+Marrison-1967-909-517	517	noon	12	MIDDAY	2
+Marrison-1967-909-518	518	north	1598	NORTH	2
+Marrison-1967-909-519	519	nose	1221	NOSE	2
+Marrison-1967-909-520	520	not	1240	NOT	2
+Marrison-1967-909-521	521	now	1376	NOW	2
+Marrison-1967-909-522	522	oath	1712	OATH	2
+Marrison-1967-909-523	523	obey	1635	OBEY	2
+Marrison-1967-909-524	524	oil	1484	OIL (ORGANIC SUBSTANCE)	2
+Marrison-1967-909-525	525	old (of persons)	2112	OLD (AGED)	4
+Marrison-1967-909-526	526	old (of things)	2113	OLD (USED)	4
+Marrison-1967-909-527	527	old man	406	OLD MAN	2
+Marrison-1967-909-528	528	old woman	907	OLD WOMAN	2
+Marrison-1967-909-529	529	on	2379	UP OR ABOVE	4
+Marrison-1967-909-530	530	onion	2366	ONION	2
+Marrison-1967-909-531	531	only	1580	ONLY	2
+Marrison-1967-909-532	532	open	1156	OPEN	2
+Marrison-1967-909-533	533	orange	2627	ORANGE (FRUIT)	2
+Marrison-1967-909-534	534	order	1128	ORDER	4
+Marrison-1967-909-535	535	orphan	1149	ORPHAN	2
+Marrison-1967-909-536	536	other	197	OTHER	2
+Marrison-1967-909-537	537	otter	15	OTTER	2
+Marrison-1967-909-538	538	outside	762	OUTSIDE	2
+Marrison-1967-909-539	539	owl	735	OWL	2
+Marrison-1967-909-540	540	owner	1036	OWNER	2
+Marrison-1967-909-541	541	pain	1783	PAIN	2
+Marrison-1967-909-542	542	palm (hand)			4
+Marrison-1967-909-543	543	palm (of hand)	1183	PALM OF HAND	2
+Marrison-1967-909-544	544	panji (bamboo stake)			
+Marrison-1967-909-545	545	pass by	558	PASS BY	2
+Marrison-1967-909-546	546	pay	718	PAY	2
+Marrison-1967-909-547	547	peace	1338	PEACE	2
+Marrison-1967-909-548	548	peacock			
+Marrison-1967-909-549	549	pearl			
+Marrison-1967-909-550	550	pen	794	PEN	2
+Marrison-1967-909-551	551	people	789	PEOPLE	2
+Marrison-1967-909-552	552	person	683	PERSON	2
+Marrison-1967-909-553	553	pierce	398	PIERCE	2
+Marrison-1967-909-554	554	pig (domestic)	1337	PIG	4
+Marrison-1967-909-555	555	pig (wild)	3154	WILD BOAR	4
+Marrison-1967-909-556	556	pig(wild)			4
```

Comment:
see above: they have two pig (wild), so link both to WILD BOAR, it doesn't make sense otherwise.